hw9/hw9_1.py

```diff
@@ -0,0 +1,23 @@
+def my_fib_func(n):
+    x = 0
+    y = 1
+    for j in range(n):
+        yield x
+        x, y = y, x + y
+
+
+result = None  # Otherwise python annoy with notification 'Name can be undefined'. And this is only way I can fix that
+while True:
+    index = input('Enter the index of the required Fibonacci number\n')
+    if index.isdigit():
+        if int(index) > 0:
+            for result in my_fib_func(int(index)):
```

Comment:
It is not necessary to use loop here. You can save your generator to some variable and transform it to list and get the last element of the list, like this: `result = list(my_fib_func(int(index)))[-1]`